client/components/ModalSeparator/ModalSeparator.tsx

```diff
@@ -0,0 +1,7 @@
+import React, { FC } from 'react';
```

Comment:
if that component deserves to exist, please keep all components in the same place (fuselage)

btw I couldn't find where you are using the component

Reply:
This is the component, in our next steps, I'll change to fuselage side.

![image](https://user-images.githubusercontent.com/2493803/111842169-c8073e80-88dd-11eb-830b-8256fbb8fb04.png)